onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.py

```diff
@@ -24,8 +34,11 @@
 library_files_to_load = [
     "onnxruntime_pybind11_state.so",
     "libonnxruntime_providers_shared.so",
-    "libonnxruntime_providers_rocm.so",
 ]
+if "CUDAExecutionProvider" in available_providers:
+    library_files_to_load.append("libonnxruntime_providers_cuda.so")
+if "ROCMExecutionProvider" in available_providers:
```

Comment:
> if "ROCMExecutionProvider" in available_providers:

I understand that only one of these will be present in `available_providers`. I'd still suggest using an `elif` to make it obvious to a reader.

Reply:
I think this is correctly and every other place that used `elif` is actually wrong as they rely on the implicit fact that CUDA EP and ROCm EP is mutally exclusive. We might not be true in the future =)